### ISSUE TITLE
IBM COS Plugin v2.2.23 libcrypto.so.1.1 Error

### ISSUE THREAD

**Issue by vdymna** [IBM/ibmcloud-object-storage-plugin#148]
### Environment 
Openshift cluster version: 4.13.14
Kubernetes version: v1.26.9+52589e6
IBM COS plugin version: ibm-object-storage-plugin-2.2.23
Object storage type: AWS S3

### Issue/Bug
When trying to mount a PVC backed by AWS S3 to a pod, the volume mount fails with the following error:
```
MountVolume.SetUp failed for volume "pvc-XXXXXXXXXXX" : mount command failed, status: Failure, reason: 
Error mounting volume: s3fs mount failed: s3fs: error while loading shared libraries: 
libcrypto.so.1.1: cannot open shared object file: No such file or directory
```

Example PVC:
```yaml
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
  name: pvc-test
  annotations:
    ibm.io/endpoint: 'https://s3.us-east-1.amazonaws.com'
    ibm.io/bucket: OUR_BUCKET
    ibm.io/region: us-east-1
    ibm.io/secret-name: pvc-test-accesskey
    volume.beta.kubernetes.io/storage-class: ibmc-s3fs-cos
    ibm.io/auto-create-bucket: 'false'
    ibm.io/auto-delete-bucket: 'false'
spec:
  accessModes:
    - ReadWriteMany
  resources:
    requests:
      storage: '1'
  storageClassName: ibmc-s3fs-cos
  volumeMode: Filesystem
```
###  Workaround
Uninstall the plugin and installing version 2.2.22 does resolve this issue.

**Comment by nkkashyap** [comment]
hi @vdymna 
We are working on releasing new version with fix. 
Mean time please use lower version as you have posted as workaround. Thanks

**Comment by mssachan** [comment]
@vdymna We are aware about the issue. The new helm chart version 2.2.24 with the fix will be available by 24th February 2024.

**Comment by vdymna** [comment]
Thanks @nkkashyap and @mssachan .

**Comment by Bhagyashreek8** [comment]
@vdymna This issue has been fixed in the chart release 2.2.24. Charts have been released today.
Thank you.
